.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@looker/eslint-config moving to _not_ including a default prettier and instead leveraging what's in the project specification (or prettier defaults if no per-repo config available)